index.html

```diff
@@ -296,6 +292,12 @@
     var daySpan = document.getElementById("day");
     daySpan.firstChild.textContent = dateToString(dateObj);
   }
+  function getFunnyImage(json_obj) {
```

Comment:
also, this returns an object, not an image. You might want to make this function just return the right random URL, and let the calling code build the object.